translations/harbour-berlin-vegan-de.ts

```diff
@@ -275,5 +275,10 @@
         <source>closes soon</source>
         <translation>schließt bald</translation>
     </message>
+    <message id="id-venue-please-tell-waiter">
+        <source>Please tell the waiter/owner that you found their venue via the Berlin-Vegan app. The significance of being listed in a database like this is often underestimated. Thanks for the help!</source>
+        <oldsource>Please tell the waiter/owner that you found their venue via the Berlin-Vegan app.</oldsource>
+        <translation>Bitte teile der/dem Kellner(in)/Inhaber(in) mit, dass du ihr Lokal über die Berlin-Vegan-App gefunden hast. Die Wichtigkeit, in Datenbanken wie dieser gelistet zu sein, wird von vielen unterschätzt. Danke für die Hilfe!</translation>
```

Comment:
> The articles should be the other way around: "dem/der Kellner(in)/Inhaber(in)"

I can do that, but why is it better / more correct?

For simplicity sake I could also use the female form only: "der Kellnerin/Inhaberin"

> Berlin-Vegan App instead of Berlin-Vegan-App?

Normally, the German language does not know about a term that exceeds words separated by spaces: http://www.spiegel.de/kultur/zwiebelfisch/zwiebel-fisch-dem-wahn-sinn-eine-luecke-a-333774.html

So I'd rather not like to make that change.

Reply:
> I can do that, but why is it better / more correct?
With our version I would read: der Kellner/Inhaber, dem Kellnerin/Inhaberin in my head, but maybe it is just me.

Reply:
> So I'd rather not like to make that change.

It is again this weird mixup: "Berlin-Vegan Guide", "Berlin-Vegan-Guide". Now we have: "Berlin-Vegan-App". I get your point, but for me this is somewhat awefull.

Reply:
So how about: `dass du ihr Lokal über den Berlin-Vegan-Guide gefunden hast.` ?

Reply:
> > So I'd rather not like to make that change.
> 
> It is again this weird mixup: "Berlin-Vegan Guide", "Berlin-Vegan-Guide". Now we have: "Berlin-Vegan-App". I get your point, but for me this is somewhat awefull.

This Berlin-Vegan / Berlin vegan guide / ... name mix, its a total mess :(

Reply:
> > I can do that, but why is it better / more correct?
> 
> With our version I would read: der Kellner/Inhaber, dem Kellnerin/Inhaberin in my head, but maybe it is just me.

Na, you are right. It is totally ugly. I was honestly considered (and still am) going for the female or even the male version only, because this is confusing. Or if you have a better idea: Please let me know.